app/services/cart.js

```diff
@@ -1,20 +1,35 @@
 import Ember from 'ember';
 
 export default Ember.Service.extend({
-  addToCart(title, author, price) {
-    //get the existing cart and add the book, or if there's not a cart then create a new cart with the book in it
-    let cart = JSON.parse(localStorage.getItem('cart'));
-    if (!cart) {
-      cart = [];
-    } 
-    cart.push({ title, author, price });
-    localStorage.setItem('cart', JSON.stringify(cart));
 
+  init() {
+  // get the cart our of local storage (this way it persists in the user's browser past a reload)
+  this.set('cart', JSON.parse(localStorage.getItem("cart")));
```

Comment:
setting this as `items` or `books` instead of `cart` gives a bit more meaning, makes code easier to read too